src/styles/components/_component-grid.scss

```diff
@@ -1,13 +1,26 @@
 $breakpoint-tablet: 820px;
 
-.components-grid {
+.grid-container {
   width: 100%;
-  text-align: center;
-  padding: 1rem;
 
   @include background-pattern;
+}
+
+.components-grid {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: auto;
+  padding: 1rem;
+  padding-top: 4rem;
+  gap: 1rem;
+  width: 100%;
+  max-width: 100rem;
+  overflow-x: hidden;
+  overflow-y: hidden;
 
   &__item {
+    border: 3px solid black;
```

Comment:
We should probably use the color map for this, although it will be replaced by the dark mode changes, so maybe fine to leave as-is.